docs/deployment/landing_page.rst

```diff
@@ -129,6 +171,19 @@ SecureDrop-related behavior.
 Instead, we recommend including .onion addresses in plain text, without a
 hyperlink.
 
+If you have been provided a short onion name for your instance, this address
+will also need to be plain text, without a hyperlink. We recommend using the
+text below to provide maximum clarity: ::
+
+    The SecureDrop instance can be found by entering the following address
+     in Tor browser: <short onion address>
```

Comment:
We generally recommend being explicit about being desktop vs. mobile (onion names won't work in the mobile app, and there are other reasons not to access SecureDrop from a mobile device), and we've standardized on the title case version of the application name. So I would say here: "the desktop version of Tor Browser".